src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,52 @@
+package duke;
+
+import java.nio.file.Paths;
+import java.nio.file.Path;
+
+/**
+ * Main class of the Duke chat-bot.
+ */
+public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constructor for the Duke class.
```

Comment:
Our coding standards state that there should be an empty line between the description and parameter section.